### ISSUE TITLE
Unable to write to gbspack.exe and unable to change tmp directory

### ISSUE THREAD

**Issue by zenzoa** [chrismaltby/gb-studio#898]
**Describe the bug**
After being able to build my game several times, after adding a new large sprite, I got an unknown build error. I removed that sprite, restarted my computer, and now I get this error:
`Error: Could not write file C:\tmp/_gbstools/gbspack/gbspack.exe: Error: ENOENT: no such file or directory, open 'C:\Users\[USERNAME]\AppData\Local\Temp\9318dae1-5433-4c0b-b0e0-c1a34b51e4c2.tmp.exe'`

I get the same error if I start a new project with one of the base templates provided.

I tried making everything in the tmp/_gbstools folder readable/writable, and I tried changing the temp folder in Preferences. However, after closing the Preferences window and re-opening it, it had returned to C:\tmp (the default).

**Expected behavior**
I expect my game to compile or give me instructions for how to fix the error, and I expect to be able to change the TEMP directory.

**Platform (please complete the following information):**
 - OS: Windows 11
 - App version 3.0.1

**Comment by bigjawed** [comment]
I have the same bug as well. It used to work at first, but now I am unable to playtest anything.

**Comment by bigjawed** [comment]
Although, I noticed my anti-virus deleting an .exe file of the same name. Maybe it's your anti-virus?

**Comment by zenzoa** [comment]
Yes, it appears to be Windows Security blocking the various temp files that get created. I'm still messing around with various exemptions to get things to compile successfully, but it looks like I might have to exempt the entire AppData\Local\Temp folder which doesn't seem particularly great. I'm also still unable to change GB Studio's tmp folder.

**Comment by zenzoa** [comment]
Since half this bug is a dupe, anyone mind if I rename it to just cover the tmp directory issue? Where the directory reverts to the default whenever I close the Preferences window?
